Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1208,6 +1208,111 @@ def get_incidents_command(client, args):
+        'CommandLine': alert.get(f'{process_type}_process_command_line'),
+        'Path': alert.get(f'{process_type}_process_image_path'),
+        'Start Time': alert.get(f'{process_type}_process_execution_time'),
+        'Hostname': alert.get('host_name')
+    }
+
+    remove_nulls_from_dictionary(process_context)
+
+    # If the process contains only 'HostName' , don't create an indicator
+    if len(process_context.keys()) == 1 and 'Hostname' in process_context.keys():
+        return {}
+    return process_context
+
+
+def add_to_ip_context(alert, ip_context):
+    action_local_ip = alert.get('action_local_ip')
+    action_remote_ip = alert.get('action_remote_ip')
+    if action_local_ip:
+        ip_context.append(
+            {
+                'Address': action_local_ip
+            }
+        )
+
+    if action_remote_ip:
+        ip_context.append(
+            {
+                'Address': action_remote_ip
+            }
+        )
+
+
+def create_context_from_network_artifacts(network_artifacts, ip_context):
+    domain_context = []
+
+    if network_artifacts:
+        for artifact in network_artifacts:
+            domain = artifact.get('network_domain')
+            if domain:
+                domain_context.append(
+                    {
+                        'Name': domain
+                    }
+                )
+
+            network_ip_details = {
+                'Address': artifact.get('network_remote_ip'),
+                'GEO.Country': artifact.get('network_country')
+            }
+
+            remove_nulls_from_dictionary(network_ip_details)
+
+            if network_ip_details:
+                ip_context.append(network_ip_details)
+
+    return domain_context
+
+
+def get_indicators_context(incident):
+    file_context: List[Any] = []
+    process_context: List[Any] = []
+    ip_context: List[Any] = []
+    for alert in incident.get('alerts'):
+        # file context
+        file_details = {
+            'Name': alert.get('action_file_name'),
+            'Path': alert.get('action_file_path'),
+            'SHA265': alert.get('action_file_sha256'),
+            'MD5': alert.get('action_file_md5')
+        }
+        remove_nulls_from_dictionary(file_details)
+
+        if file_details:
+            file_context.append(file_details)
+
+        # process context
+        process_types = ['actor', 'os_actor', 'causality_actor', 'action']
+        for process_type in process_types:
+            single_process_context = get_process_context(alert, process_type)
+            if single_process_context:
+                process_context.append(single_process_context)
+
+        # ip context
+        add_to_ip_context(alert, ip_context)
+
+    network_artifacts = incident.get('network_artifacts', [])
+
+    domain_context = create_context_from_network_artifacts(network_artifacts, ip_context)
+
+    file_artifacts = incident.get('file_artifacts', [])
+    for file in file_artifacts:
+        file_context.append({
```

Comment:
why there isn't a validation on the value before appending to the list as you do in the previous loop?